### PR TITLE
Feature/community

### PR DESCRIPTION
**Pull request by ghost**
In order to apply [community standarts](https://github.com/SparkEquation/e2e-behaviour-testing/community) I've created a template for description, updated readme, created an example of contributing guide and copied two template issues into separate files.   
We need to actualize the information for the project and instead of merging all the files do the following:
- Update description of the project
- Update project's issues templates
- Remove description and *template files from the pull request
- Decide where to put CONTRIBUTING.md file

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+email, or any other method with the owners of this repository before making a change. 
+
+## Additional dependencies
+Before adding any additional dependencies into this project you should be
+absolutely sure that we need it. Also, you must justify it's usage.  
+Please, make sure that you do not violate any legal rights
+before adding licensed code into this project.  
+Use `npm run checkLicense` to get the full list of licenses  
+
+## Style guide
```
Comment (OlegAtSpark) [inline docs/CONTRIBUTING.md:13]: I think that in this section it will be enough to place a link to eslint config in order not to duplicate the description of the rules to avoid the situation when rules here are out of sync with eslint config.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+how it affects an existing software
+- Do not include unrelated changes.
+- You should not change or omit git hooks before submitting pull request 
+- If you are asked to make changes to your pull request, do not create a new pull request
+just add more commits into an existing one.
+
+## Issues rules
+- Search the issue tracker before opening an issue.
+- Use a clear and descriptive title.
+- Include as much information as possible:
```
Comment (OlegAtSpark) [inline docs/CONTRIBUTING.md:30]: Maybe it would be useful to add a link to `FeatureRequestTemplate.md`. What do you think?

**Review by OlegAtSpark — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (ghost) [reply docs/CONTRIBUTING.md:13]: Okay, then we need to merge [this pull request](https://github.com/SparkEquation/e2e-behaviour-testing/pull/3) first

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (ghost) [reply docs/CONTRIBUTING.md:30]: According to the description of this pull request, `FeatureRequestTemplate` will be removed from git and added to github directly

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # e2e-behaviour-testing
-
+Clear and scalable e2e tests written in English only
```
Comment (OlegAtSpark) [inline README.md]: I am not sure that this description is appropriate. @NikAtSpark, @DanAtSpark, do you have any ideas?

**Review by OlegAtSpark — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (NikAtSpark) [reply README.md]: Maybe just something like - Clear and scalable e2e tests written in plain English

Comment (OlegAtSpark) [reply README.md]: I like Nik's description more. @DanAtSpark, please change the description if you have no objections.

Comment (ghost) [reply README.md]: I like this description more. I've pushed a commit with those changes

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (OlegAtSpark) [reply docs/CONTRIBUTING.md:13]: @DanAtSpark, This [PR](https://github.com/SparkEquation/e2e-behaviour-testing/pull/3) is merged. So you can add the link.

Comment (ghost) [reply docs/CONTRIBUTING.md:13]: @OlegAtSpark Done

**Review by OlegAtSpark — APPROVED** [review]
